test/HTMLView-test.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
```

Comment:
This test captures all of the previous tests, but uses [enzyme](https://github.com/airbnb/enzyme), an awesome testing library that provides a great API on top of shallow rendering.